playground/fastify/src/server.ts

```diff
@@ -1,4 +1,6 @@
-import 'dotenv/config';
+import * as dotenv from 'dotenv';
+dotenv.config();
```

Comment:
Just fyi, I expect this is going to break again because of the eslint/prettier rules we use.

Once formatted, `dotenv.config();` will be moved under the clerk import so clerk will initialise without the env variables.

Shall we move it to a `./loadEnv` file and import that instead?